docs-src/concepts/what-is-temporal.md

```diff
@@ -8,7 +8,7 @@ tags:
   - explanation
 ---
 
-Temporal is a scalable and reliable runtime for Reentrant Processes called [Temporal Workflow Executions](/concepts/what-is-a-workflow-execution).
+Temporal is a scalable and reliable runtime for durable Reentrant Processes called [Temporal Workflow Executions](/concepts/what-is-a-workflow-execution).
```

Comment:
"for durable execution." would be in line with the new branding. would want to explain or link to an explanation of what it means though